### ISSUE TITLE
Ability to configure model persistence at a per-model level

### ISSUE THREAD

**Issue by ignacio-mb** [metabase/metabase#40514]
**Is your feature request related to a problem? Please describe.**
You might have a use case where different tables update with very different periods of time, and you still want to maintain performance with model caching.

**Describe the solution you'd like**
Ability to set different caching configurations for different at a model level and not only globally

**Describe alternatives you've considered**
Disabling caching, but that is not good for certain use cases.

**How important is this feature to you?**
Requested by a customer, with the following words:

>  I'm talking about the model refresh schedule found in the admin section. We have some monitoring data on one hand (for instance to check for service health) that we would like to refresh often, for instance every 5 to 15min but on the other hand we also have many technical report (some of which take a long time to refresh) that we would like to refresh only once per day for instance. More broadly, we use Metabase as a central data hub where all the data from different team is available but different team have different definition of up to date for their models, in some case 1 day is up to date enough, in others 15 min is preferred.

**Additional context**
Internal ticket [25798](https://metabase.zendesk.com/agent/tickets/25798)

**Comment by darksciencebase** [comment]
cc @luizarakaki

**Comment by zhouzian** [comment]
I found this feature request useful as not all model cache need to be refreshed at the same schedule. Some models are aggregated per week or month, so that we only need to refresh them once a week/month. Other models may need hourly or daily refresh.

**Comment by paoliniluis** [comment]
This is also a performance issue, as refreshing all models at the same time can have some very nasty consequences on the DW

**Comment by thebiglabasky** [comment]
Duplicate of #48743 ?

**Comment by ignacio-mb** [comment]
Yep. This was opened before but it's the same. So closing in favor of that one
